packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -124,13 +136,12 @@ const ActiveVisitsTable = (props) => {
                 </TableRow>
               </TableHead>
               <TableBody>
-                {rows.slice(lowerBound, upperBound).map((row, ind) => (
+                {rows.map((row, ind) => (
                   <TableRow key={row.id} style={{ height: desktopView ? '2rem' : '3rem' }}>
                     {row.cells.map((cell) => (
                       <TableCell key={cell.id}>
                         {cell.info.header === 'name' ? (
-                          <ConfigurableLink
-                            to={`\${openmrsSpaBase}/patient/${activeVisits[lowerBound + ind].patientUuid}/chart/`}>
+                          <ConfigurableLink to={`\${openmrsSpaBase}/patient/${results[ind]?.patientUuid}/chart/`}>
```

Comment:
Isn't `results[index]` the same as `row`? Why not use `row`?

Reply:
Because `row` contains those values of the headers. As you can see activeVisits have a property patientUuid, but since there's no header for patientUuid, hence value of patientUuid doesn't exist in `row`.

Reply:
Alright - thanks for the clarification.

Alternatively, you could have put the `patientUuid` in a hidden column; but I think that way is also fine. Works for me!